build/metadata_utilities.py

```diff
@@ -248,7 +279,8 @@ def test_add_all_metadata_simple():
                         'mechanism': 'fixed',
                         'value': 1
                     },
-                    'type': 'ViSession'
+                    'type': 'ViSession',
+                    'library_call_name': 'self.vi',
```

Comment:
What is this supposed to be? The name is not descriptive.

Reply:
This is the name used for the call into the library (DLL)